gradle.properties

```diff
@@ -15,10 +15,7 @@ nolij = Nolij (@xdMatthewbx#1337)
 mod_url = https://modrinth.com/mod/zume
 repo_url = https://github.com/Nolij/Zume
 issue_url = https://github.com/Nolij/Zume/issues
-
-# JAR compression settings
-strip_lvts = true
-strip_source_files = true
+release_channel = release
```

Comment:
This shouldn't be here in the final version. It's meant to be passed as a CLI argument.

Reply:
Yes, I will remove it before release.